crates/sui-rosetta/docker/sui-rosetta-devnet/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:latest AS chef
+WORKDIR sui
+ARG GIT_REVISION
+ENV GIT_REVISION=$GIT_REVISION
+RUN apt-get update && apt-get install -y build-essential libssl-dev pkg-config curl cmake clang git
+RUN curl https://sh.rustup.rs -sSf | bash -s -- -y
+ENV PATH="/root/.cargo/bin:${PATH}"
+
+FROM chef AS builder
+RUN git clone https://github.com/MystenLabs/sui .
+# We can change this to `git checkout -b devnet` after next devnet update
+RUN git checkout 4aba20671f419257c8717be62efdc07aeb667908
+
+RUN curl -sSfL https://raw.githubusercontent.com/coinbase/rosetta-cli/master/scripts/install.sh | sh -s
+RUN curl -fLJO https://github.com/MystenLabs/sui-genesis/raw/main/devnet/genesis.blob
+RUN cargo build --release --bin sui --bin sui-rosetta
+
+# Production Image
+FROM ubuntu:latest AS runtime
+WORKDIR sui
+COPY --from=builder /sui/target/release/sui /usr/local/bin
+COPY --from=builder /sui/target/release/sui-rosetta /usr/local/bin
+COPY --from=builder /sui/bin/rosetta-cli /usr/local/bin
+COPY --from=builder /sui/crates/sui-config/data/fullnode-template.yaml /sui/devnet/fullnode.yaml
+COPY --from=builder /sui/genesis.blob /sui/devnet/genesis.blob
+RUN /usr/local/bin/sui genesis
+
+ARG BUILD_DATE
+LABEL build-date=$BUILD_DATE
+LABEL git-revision=$GIT_REVISION
```

Comment:
Do you want to add a command here so that people who `docker run` this container will automatically get the rosetta server running?

Reply:
this image is being used to start the rosetta-online or rosetta-offline server, user will need to specify which server they want to start, ideally they should use the docker-compose file but they can also run `docker run mysten/sui-rosetta-local sui-rosetta start-offline-server` if they don't use docker-compose

Reply:
ok sounds good, is that docker run command line documented anywhere?

Reply:
I have added the docker run instruction to readme.md